platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/multi_project_builds.adoc

```diff
@@ -134,31 +278,72 @@ Gradle will then look for the build file for the new subproject in the `lib/` su
+[source]
+----
+dependencies {
+    implementation(Conventions.kotlinStdLib)
+}
+----
+
+[[sub:modifying_element_of_the_project_tree]]
+== Modifying elements
+
+The multi-project tree created in the settings file comprises _project descriptors_.
+
+You can modify these descriptors in the settings file at any time.
+
+To access a descriptor, you can:
+
+====
+include::sample[dir="snippets/multiproject/customLayout/kotlin",files="settings.gradle.kts[tags=lookup-project]"]
+include::sample[dir="snippets/multiproject/customLayout/groovy",files="settings.gradle[tags=lookup-project]"]
 ====
 
+Using this descriptor, you can change the name, project directory, and build file of a project:
+
+====
+include::sample[dir="snippets/multiproject/customLayout/kotlin",files="settings.gradle.kts[tags=change-project]"]
+include::sample[dir="snippets/multiproject/customLayout/groovy",files="settings.gradle[tags=change-project]"]
+====
 
-Next, will explore how build logic can be <<sharing_build_logic_between_subprojects#sharing_build_logic_between_subprojects,shared between subprojects>>
-and how subprojects can <<declaring_dependencies_between_subprojects#declaring_dependencies_between_subprojects,depend on one another>>.
+Consult the link:{javadocPath}/org/gradle/api/initialization/ProjectDescriptor.html[ProjectDescriptor] class in the API documentation for more information.
 
+[[sec:naming_recommendations]]
 == Naming recommendations
 
-As your project grows, naming and consistency gets increasingly more important.
+As your project grows, naming and consistency get increasingly more important.
 To keep your builds maintainable, we recommend the following:
 
-1. _Keep default project names for subprojects_:
+1. *Keep default project names for subprojects*:
 It is possible to configure custom project names in the settings file.
-However, it’s an unnecessary extra effort for the developers to keep track of which project belongs to what folders.
-
-2. _Use kebab case formatting for all project names_:
-A kebab case formatting is when all letters lowercase, words separated with a dash (‘-’) character (e.g.`kebab-case-formatting`).
-This is already the de-facto pattern for many large projects.
-Besides, Gradle supports <<command_line_interface.adoc#sec:name_abbreviation,name abbreviation for kebab case names>>.
+However, it’s an unnecessary extra effort for the developers to track which projects belong to what folders.
 
-3. _Define the root project name in the settings file_:
-The ´rootProject.name´ effectively assigns a name to the build as a whole, which is used in reports like build scans.
-If the root project name is not set, the name will be the container directory name, which can be unstable (i.e. you can check out your project to any directory).
-If the root project name is not set and it is checked out to a root of a file system (e.g. `/` or `C:\`), the name will be generated randomly.
+2. *Use kebab case formatting for all project names*:
```

Comment:
💭 I thought we had previously been told that we shouldn't say "kebab case" any more, and should replace it with "lower hypenated case" or something?  We should update this section if so.

Reply:
Old docs, thanks for catching this. I updated it.